server/src/main/java/org/exoplatform/chat/server/ChatServer.java

```
@@ -533,6 +534,101 @@ else if (targetUser.startsWith(ChatService.TEAM_PREFIX))
     }
     return Response.ok("Updated!");
   }
+  
+  @Resource
+  @MimeType("text/plain")
+  @Route("/isFavorite")
+  public Response.Content isFavorite(String user, String token, String targetUser, String dbName) {
+    if (!tokenService.hasUserWithToken(user, token, dbName)) {
+      return Response.notFound("Petit malin !");
+    }
+    boolean isFavorite = false;
+    try {
+      isFavorite = userService.isFavorite(user, targetUser, dbName);
+    } catch (Exception e) {
+      LOG.log(java.util.logging.Level.WARNING, e.getMessage());
```

Do not swallow the exception.